fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1804,6 +1805,7 @@ primaryExpression
     | CASE caseExpr=expression whenClause+ (ELSE elseExpression=expression)? END          #simpleCase
     | CASE whenClause+ (ELSE elseExpression=expression)? END                              #searchedCase
     | arrayType? '[' (expressionList)? ']'                                                #arrayConstructor
+    | mapType? '{' (expressionList)? '}'                                                  #mapConstructor
```

Comment:
```suggestion
    | mapType? '{' (expressionList)? '}'                                                  #mapConstructor
```
if we support constant json in the future, how to distinguish between the syntax of json?

Reply:
json should be '{}' ? 
trino/hive generate json should use functions, CK should use the format of '{}'

Reply:
maybe conflict with struct?

Reply:
> json should be '{}' ? trino/hive generate json should use functions, CK should use the format of '{}'

yeah, just a discussion